src/python/grpcio/grpc/_grpcio_metadata.py

```diff
@@ -1,17 +1 @@
-# Copyright 2017 gRPC authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-# AUTO-GENERATED FROM `$REPO_ROOT/templates/src/python/grpcio/grpc/_grpcio_metadata.py.template`!!!
-
-__version__ = """1.29.0.dev0"""
+__version__ = """1.29.0.dev0"""
```

Comment:
Why this change?

Reply:
not sure what actually changed :/

Reply:
ohh it somehow got rid of the headers. i'm not sure why this file got regenerated/touched